openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -52,8 +52,9 @@ def process(self, context):
         for instance in context:
             # Check if instance is a review by checking its family
             # Allow a match to primary family or any of families
-            families = set([instance.data["family"]] +
-                           instance.data.get("families", []))
+            families = set(
```

Comment:
I feel like something like this is more readable:
```py
families = set(
    instance.data.get("families, []).append(instance.data["family"])
)
```

Reply:
Not only is the old way more readable. The proposed way wouldn't even work because `append` doesn't return anything. Right?

If it however did work then you'd be editing the `families` list in place which is also NOT what we want. So I'd say the old logic is correct and perfectly readable to me.

If it's really currently unreadable to many I guess we could turn it into:
```python
families = set(instance.data.get("families", []))
families.add(instance.data["family"])
```

But I don't see a direct reason to do so.

Reply:
Your example code doesn't work if `instance.data.get("families", {})` actually returns a list. It'd then be:

```python
set(["a"] | {"b"})
```

Resulting in:
```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for |: 'list' and 'set'
```

You'd still need to convert the first entry to a set since `families` is a list.

Reply:
I'm sorry but that's exactly what it does :) `set` is only for "families"

Reply:
Oh you are right, I assumed the append would be casted on the output of get, and that would be returned, I stand corrected.

Reply:
> I'm sorry but that's exactly what it does :) set is only for "families"

I missed the closing `)` bracket for `set` there. Thanks for pointing it out! I'm not saying it's the most readable one but it should be functional.